code/modules/mob/living/simple_animal/hostile/retaliate/cavern.dm

```diff
@@ -113,38 +113,6 @@
 		loot += pick(/obj/item/weapon/ore/silver, /obj/item/weapon/ore/gold, /obj/item/weapon/ore/uranium, /obj/item/weapon/ore/diamond)
 		i--
 
-/mob/living/simple_animal/hostile/retaliate/minedrone/Life()
-	..()
-	FindOre()
-
-/mob/living/simple_animal/hostile/retaliate/minedrone/proc/FindOre()
```

Comment:
Instead of removing this behaviour. Could just have it delete the ore if it reaches a certain count (like 50)?

Reply:
The main issue is not the ores, it is that they just get on the way of each other and run away from the mining site, so the mining drones just walk into holes or end floating on space.